spec/sidekiq/benefits_intake_status_job_spec.rb

```diff
@@ -211,6 +211,38 @@
       # end 'updating the form submission status'
     end
 
+    describe 'updating the Burial form submission status' do
+      before do
+        Flipper.enable(:burial_received_email_notification)
```

Comment:
Please use allow(Flipper).to receive(:enabled?).with(: burial_received_email_notification, instance_of(User)).and_return(true) OR
allow(Flipper).to receive(:enabled?).with(: burial_received_email_notification).and_return(true)

Reply:
@RachalCassity Fixed!